examples/panel/weights-and-biases-llm/app.py

```diff
@@ -59,7 +67,7 @@ def wait_on_run(run, thread):
             thread_id=thread.id,
             run_id=run.id,
         )
-        time.sleep(0.5)
+        time.sleep(0.00001)
```

Comment:
why do we need this sleep?

Reply:
due to how the assistants api work 

followed their cook book

[see here
](https://cookbook.openai.com/examples/assistants_api_overview_python)